package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.16.5",
```

Comment:
![image](https://github.com/Paul-tes/Math-magicians/assets/93039359/b9c65985-c390-4c19-a80d-8819c7bd87eb)

- [ ] It would be nice if you fix the links highlighted in the image above ⬆️ to make your readme descriptive and professional 👍